src/components/NurseryWithdrawals/NurseryWithdrawalsTabContent.tsx

```diff
@@ -0,0 +1,23 @@
+import { Typography, useTheme } from '@mui/material';
+import strings from 'src/strings';
+import Card from 'src/components/common/Card';
+import NurseryWithdrawalsTable from './NurseryWithdrawalsTable';
+
+export default function PlantingProgress(): JSX.Element {
```

Comment:
withdrawals table, some style of title and missing description etc we can fix as follow up, not scope of this PR